tests/NovelRT.Tests/CMakeLists.txt

```diff
@@ -51,24 +34,53 @@ set(SOURCES
   Utilities/EventTest.cpp
 
   main.cpp
- )
+)
+
+if(NOVELRT_BUILD_INTEROP)
```

Comment:
Instead of using an if here, you should use generator expressions:
```cmake
set(sources
  ...
  $<$<BOOL:${NOVELRT_BUILD_INTEROP}>:the_file.cpp>
)
```

Reply:
OOOOOOOoooo I did not know this was a thing.